website/docs/cloud-docs/api-docs/team-tokens.mdx

```diff
@@ -85,7 +346,7 @@ curl \
   --header "Content-Type: application/vnd.api+json" \
   --request POST \
   --data @payload.json \
-  https://app.terraform.io/api/v2/teams/team-BUHBEM97xboT8TVz/authentication-token
+  https://app.terraform.io/api/v2/teams/team-BUHBEM97xboT8TVz/authentication-tokens
```

Comment:
This should be `authentication-tokens` here since it's for the legacy APIs.

Reply:
Great catch!